build.cake

```diff
@@ -221,6 +224,7 @@ Task("PackageNuGet")
 	.IsDependentOn("CreateWorkingImage")
 	.Does(() => 
 	{
+        System.Console.WriteLine("Packaging the nuget");
```

Comment:
`Information` rather than `System.Console.WriteLine`, and `"Creating the NuGet package"` since I don't think people say "the nuget"?